src/wxflow/sqlitedb.py

```diff
@@ -17,6 +17,9 @@ class SQLiteDB:
 
     """
 
+    OperationalError = sqlite3.OperationalError
+    IntegrityError = sqlite3.IntegrityError
+
```

Comment:
This works, but if you look at classes, having a:

```
class SQLiteDBError(sqlite3.Error):

OperationalError = sqlite3.OperationalError
IntegrityError = sqlite3.IntegrityError
```

Reply:
Ok, updated